lib/common/deep-equal.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const deepEqual = (x, y) => {
```

Comment:
no I just wrote it fast but we don't have any test suite at all on muon right now :( We have to fix soon.